_config.yml

```diff
@@ -147,7 +152,12 @@ t:
         title: "Atraksis"
         url: "https://atraksis.fr/"
         logo: "https://atraksis.fr/wp-content/themes/atraksis/assets/img/logo-atraksis-footer.svg"
-        content: "Our partner for field testing phases, and our intermediary with firefighters."
+        content: "An innovation player for rescue services, and our intermediary with firefighters."
```

Comment:
"A stakeholder for rescue innovation, and our point of contact with firefighters."